src/encoded/static/components/regulome_search.js

```diff
@@ -587,6 +589,7 @@ const Facet = (props) => {
                         buttonLabel={d}
                         facetLabel={facetName}
                         addGenomeFilter={addGenomeFilter}
+                        key={facetName}
```

Comment:
…except you need to have something unique for each loop iteration here, so something unique based on each element of `facetArray` should fix the issue. `facetName` is the same for every iteration, so this I think would normally generate a console error.